AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -225,10 +224,22 @@ + (BOOL)hasCrashedInLastSession {
   return [[MSCrashes sharedInstance] didCrashInLastSession];
 }
 
++ (void)setHasCrashedInLastSession:(BOOL)hasCrashedInLastSession {
+  [[MSCrashes sharedInstance] setDidCrashInLastSession:hasCrashedInLastSession];
+}
+
 + (BOOL)hasReceivedMemoryWarningInLastSession {
   return [[MSCrashes sharedInstance] didReceiveMemoryWarningInLastSession];
 }
 
++ (void)setHasReceivedMemoryWarningInLastSession:(BOOL)hasReceivedMemoryWarningInLastSession {
```

Comment:
This one (and a few others) should be `readonly`. Adding setter is incorrect here

Reply:
Fixed by 2df5e7466e9a17a6b622a4e909666598245700dd.